java/src/jmri/jmrix/loconet/locomon/Bundle.java

```diff
@@ -12,7 +12,7 @@
 
 @javax.annotation.concurrent.Immutable
 
-/**
+/*
```

Comment:
Is this change deliberate? Just checking.

Reply:
dangling jdoc